actions/setup/action.yml

```diff
@@ -95,7 +101,18 @@ runs:
       username: ${{ github.actor }}
       password: ${{ inputs.github_token }}
   - name: Setup Remote Runner
-    uses: input-output-hk/catalyst-ci/actions/configure-runner@master
+    uses: input-output-hk/catalyst-ci/actions/configure-runner@fix/ci-cli-build
     if: ${{ inputs.earthly_runner_secret != '' && inputs.earthly_skip_install != 'true' }}
     with:
       secret: ${{ inputs.earthly_runner_secret }}
+  # CI CLI is set to 'local', run release to create ci artifact
+  # use buildkit-host tag
+  - name: Build CI CLI
+    uses: input-output-hk/catalyst-ci/actions/run@fix/ci-cli-build
+    if: ${{ inputs.cli_skip_install == 'false' && inputs.cli_version == 'local' }}
+    with:
+      artifact_name: 'ci'
```

Comment:
In the original form, you should be able to set the `artifact` to a temp directory and then manually copy the resulting binary to `/usr/local/bin/` in a subsequent step.